cheroot/workers/threadpool.py

```diff
@@ -7,6 +7,7 @@
 import threading
 import time
 import socket
+import select
```

Comment:
How about using epoll? https://github.com/cherrypy/cherrypy/issues/769#issuecomment-274639308

Reply:
epoll is only supported on Linux

Reply:
Yeah, so it'd need fallbacks